src/components/Stylebar.tsx

```diff
@@ -27,23 +33,42 @@ const Stylebar = (props: {
 
   return (
     <div className={`stylebar visibility-${props.visibility}`}>
-      {actions.map((action) => (
-        <OverlayButton action={action} callback={props.doAction} key={action} />
-      ))}
-      <StyleMenu currentStyle={props.currentStyle} doAction={props.doAction} />
-      {mobileMethods.map((action) => (
-        <OverlayButton
-          action={
-            action === Action.FullScreen
-              ? !isFullscreen
-                ? Action.EnterFullScreen
-                : Action.ExitFullScreen
-              : action
+      <input
+        accept=".json"
+        onChange={(e) => props.openFile(e.target.files)}
+        multiple={true}
+        ref={fileInputRef}
+        type="file"
+      />
```

Comment:
Just use the existing drop image functionality and put json stuff in processFiles?

Reply:
uh i dont understand the drop image functionality fully so if you want to do that you can

Reply:
https://github.com/cjquines/qboard/issues/39